source/simulator/assemblers/stokes.cc

```diff
@@ -30,6 +31,22 @@ namespace aspect
     template <int dim>
     void
     StokesAssembler<dim>::
+    create_additional_material_model_outputs(MaterialModel::MaterialModelOutputs<dim> &outputs) const
+    {
+      if (this->get_parameters().enable_additional_stokes_rhs
+          && !outputs.template get_additional_output<MaterialModel::AdditionalMaterialOutputsStokesRHS<dim> >())
```

Comment:
I'll keep fighting against using `operator !` to compare pointers for non-Null-ness till I die! :-)

Please write the second condition as 
```
         && (outputs.template get_additional_output<MaterialModel::AdditionalMaterialOutputsStokesRHS<dim> >() != NULL)
```
or similar.